mne/cov.py

```diff
@@ -338,10 +339,20 @@ def _check_n_samples(n_samples, n_chan):
         logger.warning(text)
 
 
+@deprecated('"compute_raw_data_covariance" is deprecated and will be '
+            'removed in MNE-0.11. Please use compute_raw_covariance instead')
 @verbose
 def compute_raw_data_covariance(raw, tmin=None, tmax=None, tstep=0.2,
                                 reject=None, flat=None, picks=None,
                                 verbose=None):
+    return compute_raw_covariance(raw, tmin, tmax, tstep,
+                                  reject, flat, picks, verbose)
```

Comment:
you don't actually need to pass `verbose` here, but I'll let it slide since this is deprecated anyway :)